src/event-notion/NotionEvent.ts

```diff
@@ -463,7 +479,9 @@ export default class NotionEvent {
     this.youtubeLink = null;
     this.prRequests = formResponse['Any additional comments or requests?'];
     this.avEquipment = this.recording === 'Yes' ?  'From Venue' : 'N/A';
+    this.techRequests = formResponse['If you need tech or equipment, please specify here'];
```

Comment:
If this field is not mandatory in the Host Form (you should ask Events about that), you should account for that by adding "null" as a possible type in the NotionEvent class and checking if this field is null before uploading (Notion API refuses to take empty strings for properties).

Reply:
I think uploadToNotion already makes an empty string check on this.techRequests before deciding to add it or not. Is that enough or should I add a null check too?

Reply:
I’m pretty sure that, by default, the formResponse value is not null, but an empty string, so you’ll want to do something like:

```ts
this.techRequests = formResponse["If you need tech or equipment, please specify here"] !== "" ? formResponse["If you need tech or equipment, please specify here"] : null;
```

Should work along with the check you already added.

Reply:
Oh I was saying that if it was left blank, this.techRequests would be an empty string by default and not get added because of the check
`...(this.techRequests !== '' ? {`
`          'Tech Requests': {`
`            rich_text: toNotionRichText(this.techRequests),`
`          },`
`        } : {}),`
I make in line 658. I went this way because I saw similar formatting for the other request fields when they were being uploaded.

Reply:
Sorry, you are quite correct. This is fine.

Just make sure to check out the extra Property in the schema.